pyne/alara.py

```diff
@@ -1015,6 +1014,25 @@ def _output_flux(ve, tag_flux, output, start, stop, direction):
     output += u"\n\n"
     return output
 
+def _output_flux_block(ve, tag_flux, reverse):
```

Comment:
Should we also delete `_output_flux()`? It doesn't appear to be used anymore.

Reply:
Yes, done in https://github.com/pyne/pyne/pull/1397/commits/a7a2e0c82e21dab97556845510c3ded4043fc140

Reply:
Test results:
| num_ves | 4 | 100 | 1000 | 10000 |
|--- | --- | --- | --- | --- |
|join then write, time (s) | x | x | x | x |
| write block by block (s) | x | x | x | 1.44 |